src/chains/ethereum/ethereum/tests/api/eth/feeHistory.test.ts

```diff
@@ -0,0 +1,525 @@
+// @ts-nocheck
```

Comment:
What are you hiding?!? :rofl: 

Haha, I don't love this, personally.

Reply:
Sometimes you just need to be weird, yah know? I think I just wanted to lay the tests down without ts barking at me.

Reply:
Oh, I remember why -> I ::boggled:: over this 
![image](https://user-images.githubusercontent.com/6285466/188963739-b49fb85e-5404-4336-8d86-94121c52ce67.png)

Reply:
:+1: It gone